module/assessments/helper.js

```diff
@@ -422,9 +424,129 @@ module.exports = class AssessmentsHelper {
                 "externalId",
                 "name",
                 "description",
-                "imageCompression"
+                "imageCompression",
+                "isAPrivateProgram"
             ]);
         })
     }
 
+      /**
+   * Assessment meta form.
+   * @method
+   * @name metaForm
+   * @param {String} solutionId - Assessment solution id.
+   * @param {String} subType - can be individual or institutional.
+   * @returns {JSON} - Form details.
+   */
+
+  static metaForm(solutionId,subType) {
+
+    return new Promise(async (resolve, reject) => {
+
+        try {
+
+            let solutionsData = await solutionsHelper.solutionDocuments({
+                _id : solutionId,
+                type : messageConstants.common.ASSESSMENT,
+                subType : subType,
+                isReusable: true
+            }, [
+              messageConstants.common["ASSESSMENT_META_FORM_KEY"]
+            ]);
+
+
+            if (!solutionsData[0]._id) {
+                throw { 
+                    status: httpStatusCode.bad_request.status, 
+                    message: messageConstants.apiResponses.SOLUTION_NOT_FOUND
+                };
+            }
+
+            let filteredData = {
```

Comment:
@aman-tunerlabs - Send default form for assessment if no specific form key given in the solution document.